benchmark/smallbank/smallbankOperations.js

```diff
@@ -68,7 +68,7 @@ function generateWorkload() {
     if(workload.length === txnPerBatch) {
         return workload;
     }
-    else {
```

Comment:
Why this changes? create account operation may be part of batch, but it should be first in the order.

Reply:
Since Fabric does not support batch, txns in the same batch are submitted nearly simultaneously . There is a high chance of error if an transaction such as send_payment is wrapped in a batch with a create_account transaction and those two txns operates on the same account. The change is to avoid this.

Reply:
Is it happening always? or with higher tps rate? Intention of small bank benchmark use case is simulate the real bank use case. if order of transactions is fixed then, it should create account first and later it perform operations on those account. For amalgamate we can do perform separate testing. Other than this we shouldn't change the behavior small bank work load generation.

Reply:
It happens always. In real world, if you are using fabric, you should first create account, **waiting for the result** and then send other operations. You should not send the later operation if the creation has not been finished.

Reply:
Then it is limitation in Fabric, even if we are sending the transactions in the right order where account creation is first. In this case should we our benchmark test to fit to the limitation of hyperledger framework?

Reply:
I've summarized the current discussion below, and will post it to contributor channel of rocketchat to see if other guys have  any suggestion.